### PR TITLE
docs: README minified dist script in <script>

### PR DESCRIPTION
**Pull request by tpscrpt**
#560 on which I've left a comment

### CODE REVIEW

**Comment by tpscrpt** [comment]
https://cdn.minifyjs.xyz/blockstack/blockstack.min.js (I've created this service to automatically update based off master every hour at X:00 and to be served from many locations based on request origin location)

**Comment by aulneau** [comment]
I've made a comment on the issue you create: https://github.com/blockstack/blockstack.js/issues/560#issuecomment-442922635

File: package.json [hunk 1]
```diff
@@ -6,16 +6,18 @@
   "scripts": {
     "browserify": "./node_modules/.bin/browserify lib/index.js --standalone blockstack -o ./dist/blockstack.js",
     "browserify-tests": "./node_modules/.bin/browserify lib/index.js --standalone blockstack -o ./tests/browserTests/bundle.js",
+    "uglify": "./node_modules/.bin/uglifyjs ./dist/blockstack.es5.js --compress --mangle reserved=['BigInteger','ECPair','Point'] --output ./dist/blockstack.min.js",
```
Comment (aulneau) [inline package.json]: Nice this is looking better. I would actually suggest using [terser](https://github.com/terser-js/terser) as uglify is not actively maintained anymore.

**Review by aulneau — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (tpscrpt) [reply package.json]: alright, I'll give it a shot.

**Comment by moxiegirl** [comment]
@JeremiGendron The branch has conflicts. Can you resolve them? @aulneau Also if you like, I can resolve them...if you think everything is up to snuff. Let me know.

**Comment by tpscrpt** [comment]
Sure thing

**Comment by moxiegirl** [comment]
@aulneau not sure...but you may have to do something on your end to approve a @JeremiGendron response to your review and move this...

**Comment by zone117x** [comment]
@JeremiGendron Thanks for getting this convo started and this initial PR work.

I'm closing this PR since we now have a CDN dist snippet being generated during our build process: https://github.com/blockstack/blockstack.js/blob/master/mdincludes/script-dist-file.md

**Pull request closed without merging**